### ISSUE TITLE
JavaScript export uses incorrect parameters for driver.manage().window().setRect()

### ISSUE THREAD

**Issue by oatmealdealer** [SeleniumHQ/selenium-ide#961]
## Bug Report

Exported code to JavaScript attempts to resize the window like so, passing the dimensions as positional parameters:
```javascript
await driver.manage().window().setRect(1024, 768);
```

When this code is run, it will do nothing.

However, the official WebDriver documentation at https://www.selenium.dev/documentation/en/webdriver/browser_manipulation/#set-window-size recommends the following code:
```javascript
await driver.manage().window().setRect({ width: 1024, height: 768 });
```

Which works perfectly.

## To Reproduce

The steps to reproduce the behavior are simple:
1. Export any `.side` file to JavaScript Mocha
2. Run the file

## Project file reproducing this issue

Here is a .zip of both a sample `.side` file and the exported `.spec.js` file:

[bugreport.zip](https://github.com/SeleniumHQ/selenium-ide/files/4237063/bugreport.zip)

## Environment

OS: Solus Linux
Selenium IDE Version:  3.16.1
Browser: Chrome
Browser Version: Version 80.0.3987.100 (Official Build) (64-bit)
chromedriver version: 80.0.1

**Comment by capoerafreak** [comment]
You can make a pull request and Change the file packages/code-export-javascript-mocha/src/command.js

**Comment by raju249** [comment]
@tourdedave we can close this issue as it is fixed in #1037 👍 

cc - @AutomatedTester

**Comment by diemol** [comment]
Thanks for pointing it out @raju249, closing it!
